core/client/app/services/clock.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+
+const {
+    Service,
+    run
+} = Ember;
+
+const ONE_SECOND = 1000;
+
+// Creates a clock service to run intervals.
+
+export default Service.extend({
+    second: null,
+    minute: null,
+    hour:   null,
+
+    init() {
+        this.tick();
+    },
+
+    tick() {
+        let now = moment.utc();
+        this.setProperties({
+            second: now.seconds(),
+            minute: now.minutes(),
+            hour:   now.hours()
+        });
+
+        let self = this;
```

Comment:
In order to fix the issues with the tests, the following four lines should be changed to this:

```javascript
if (!Ember.testing) {
    run.later(() => {
        this.tick();
    }, ONE_SECOND);
}
```

This fixes the issues with the test timeouts. cc @kevinansfield just to make sure this is okay behavior